src/components/Products/Products.jsx

```diff
@@ -36,11 +39,14 @@ export const Products = () => {
   );
   const sortedProducts = sortByPrice(sortBy, filteredProducts);
 
+  const ratedProducts = sortByRating(sortedProducts, rating);
```

Comment:
Here you can use the concept of composition of functions and use it here
For example: [see this](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce#function_composition_enabling_piping)